README_dev.md

```diff
@@ -128,14 +129,16 @@ npm run test:local -- testSuiteName
           "platformClientId": "...",
           "platformClientSecret": "..."
       },
-      "database": { "driver": "lokijs", "folder": "./" },
+      "database": { "driver": "lokijs", "folder": "./", "dbName": "local", "dbNamePrefix": "l" },
```

Comment:
I don't see why you changed this, at least `dbNamePrefix` is not required

Reply:
yea i once had the problem that they both where required dont know why it is working now ^^ but i will remove both, as they are already specified in the default config.